queries/lalrpop/highlights.scm

```diff
@@ -46,6 +51,31 @@
 (binding_symbol
   name: (identifier) @variable.parameter)
 
+(annotation
+  "#" @punctuation.special)
+
+(grammar_parameter
+  (identifier) @variable.parameter)
+
+(associated_type
+  (identifier) @type)
+
+(parametrized_type
+  [
+    "<"
+    ">"
+  ] @punctuation.bracket)
+
+(nonterminal_name
+  [
+    "<"
```

Comment:
seems like `<` and `>` aren't used as an operator anywhere, might as well just have a generic capture for them instead of being specific 5x

```
[
  "<"
  ">"
] @punctuation.bracket
```